src/components/ItemList.svelte

```diff
@@ -25,16 +23,20 @@
   let pagedItems;
   let paginated = true;
 
-  let currentPage = writable(1);
-  setContext("currentPage", currentPage);
+  // track change of filter input
+  let search = $pageState.search;
```

Comment:
ESLint wants us to use object destructuring here:
```suggestion
  let search = { $pageState };
```